api/src/opentrons/protocol_api/labware.py

```
@@ -458,7 +459,13 @@ def _load_definition_by_name(name: str) -> dict:
         saved to disc. The definition file must have been saved in a known
         location with the filename '${name}.json'
     """
-    raise NotImplementedError
+    labware_def: dict = {}
+    def_path = 'shared_data/definitions2/{}.json'.format(name)
+    try:
+        labware_def = json.loads(pkgutil.get_data('opentrons', def_path))  # type: ignore # NOQA
+    except FileNotFoundError:
+        print("{} definition not found".format(name))
```

This should probably be a log statement or (my preference, anyway) just let the exception bubble up to the user so we fail fast rather than ending up failing in unpredictable ways. I think it's an error if this function can't find a definition that matches the schema, which `{}` definitely doesn't.

---

ditto, I'd say let it raise the error